### ISSUE TITLE
order of hosts within a group

### ISSUE THREAD

**Issue by tolgaulas** [xpipe-io/xpipe#205]
Currently hosts are displayed in order they are created. It would nice to be able re-order them by drag & drop ideally, or by changing their order up and down..

**Comment by crschnick** [comment]
Yeah I can see that being useful, I will have to think about how to implement it.

Right now you can switch the ordering on a per-folder basis with these buttons
![image](https://github.com/xpipe-io/xpipe/assets/72509152/fa3f4da2-1d43-40a5-bbed-026eb4923cda)
between alphabetical and last-used orders, but yeah these do not apply to individual connections and do not allow for custom ordering.

**Comment by crschnick** [comment]
I thought really hard about this and implemented the most straightforward solution I came up with in the latest PTB: https://github.com/xpipe-io/xpipe-ptb

I also experimented with more sophisticated ordering options but in practice these were too complicated and too tedious. I think in practice, being able to pin certain connections covers many cases. Especially when new connections are always added, having a more fine-grained ordering isn't very fun to use.
